com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -355,6 +355,7 @@ private static CommandLine parse(String[] args) {
 
         options.addOption("p", "platform", true, "Platform (when bundling)");
         options.addOption("bo", "bundle-output", true, "Bundle output directory");
+        options.addOption("bf", "bundle-format", true, "Format of the created bundle (Android: 'apk' and 'aab')");
```

Comment:
I decided to go for a single value instead of comma separated list of formats. If you need both APK and AAB you can call the bundle step twice.